api/api/controllers/cluster_controller.py

```diff
@@ -341,239 +275,6 @@ async def get_configuration_node(node_id: str, pretty: bool = False, wait_for_co
     return response
 
 
-async def get_daemon_stats_node(node_id: str, pretty: bool = False, wait_for_complete: bool = False,
```

Comment:
We are maintaining the `GET /cluster/{node_id}/daemons/stats` but its implementation will be part of the `Dashboard revamp` objective, related to the metrics of the Server components.
We should return a `Not implemented` response.